applications/Aisland-DocSig.md

```diff
@@ -0,0 +1,241 @@
+ADC keeps a copy of the different drafts till the first signature is applied.  
+A secret link is generated to request the signing to the counterpart.  
+The workflow provide visibility on the different status of the documents: Draft, Waiting (for signing), Completed (all parties have signed), Rejected (one party rejected it).  
+
+  
+There is no e-signing documents solution in the Polkadot/Kusama echo-system, instead there are all the layers to create something better of commercial services. In the web2 world, https://docusign.com declares about 200 million paying user and > 1 billion users who used their platform. 
+
+[Aisland Dao](https://aisland.io) is an experimental micro-nation with a small territory of 14000 square feet. Aisland is based on a deck barge floating in international waters since 22 December 2022 in the Arabian Gulf.  
+The current small territory can be increased with additional barges properly interconnected.  
+The main purpose of Aisland Dao is to give full sovereignty to the individual and a real place where the wider principles of freedom are  applied.  
+The decentralized e-signing of documents is one of the many tools that every free and sovereign person should have the right to use.  
+
+
+### Project Details
+
+Preface:  
+We have already developed a minimum viable product (MVP) that is available at [https://docsig.aisland.io](https://docsig.aisland.io). The current MVP is offering the following features:
+
+- Dashboard to manage the workflow
+- Create/Edit a document on-line using the block editor (editor.js).  
+- Clone a public Template and Edit.  
+- Upload a document in different formats.
+- Sign the document.
+- Generate unique link to allow the signing of the counterpart.
+- View on-line the documents
+- Delete the document in drafts
+- Rejects the signature on proposed documents for signing.
+Here the workflow:
+![Workflow](https://github.com/aisland-dao/docsig/blob/main/img-docs/docusign-workflow.png)
+
+Here a few screenshots:
+![Dashboard](https://github.com/aisland-dao/docsig/blob/main/img-docs/docusign-dashboard.png)  
+![Operations](https://github.com/aisland-dao/docsig/blob/main/img-docs/docusign-operations.png)  
+![Blank Document](https://github.com/aisland-dao/docsig/blob/main/img-docs/docusign-blankdocument.png)
+![Signing](https://github.com/aisland-dao/docsig/blob/main/img-docs/docusign-signing.png)  
+
+  
+The project proposal consists in adding a new set of features to make a production ready version, comparable to the best WEB2 solution offering like https://docusign.com.  Here is the list of the new features to be added:  
+  
+1) Standard Signatures Management to allow the user to set his own standard signature image.  
+2) Signature Marker, to allow the placing of signatures onthe document for a visual improvement.  
+3) Document Encrytion, to manage the privacy of the documents even when saved on chain.  
+4) PDF outout of the signed agreement with veriable QR code.  
```

Comment:
What's an "outout"?

Reply:
I've fixed the typing error in:
"4) PDF output of the signed agreement with verifiable QR code. "